packages/pluggableWidgets/progress-circle-web/typings/progressbar.js.d.ts

```diff
@@ -16,7 +16,7 @@ declare namespace progressbar {
         color?: string;
         trailColor?: string;
         trailWidth?: number;
-        strokeWidth: number; // For IE & Edge support, shouldn't be over 6
+        strokeWidth?: number; // For IE & Edge support, shouldn't be over 6
```

Comment:
We don't support IE anymore. I suggest removing the "IE" part to avoid confusion

Reply:
In Edge it also just works (verified through Parallels), so I will just remove the comment alltogether